core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFormatterUtil.java

```diff
@@ -121,7 +138,9 @@ interface DateTimeFormatHandler {
           .put("%Y", (date) -> "0000")
           .put("%y", (date) -> "00")
           .put("%D", (date) -> null)
-          .put("%f", (date) -> // %f - Microseconds
+          .put(
+              "%f",
+              (date) -> // %f - Microseconds
               String.format(NANO_SEC_FORMAT, (date.getNano() / 1000)))
```

Comment:
Will SL keep this?
```suggestion
          // %f - Microseconds
          .put("%f", (date) -> String.format(NANO_SEC_FORMAT, (date.getNano() / 1000)))
```

Reply:
Not with the comment.... I put the comment on the previous line as it makes it slightly nicer.